sapi/cli/php_cli.c

```diff
@@ -557,15 +557,15 @@ static void cli_register_file_handles(void)
 	php_stream_to_zval(s_out, &oc.value);
 	php_stream_to_zval(s_err, &ec.value);
 
-	ZEND_CONSTANT_SET_FLAGS(&ic, CONST_CS, 0);
+	Z_CONSTANT_FLAGS(ic.value) = 0;
```

Comment:
Not sure if we should do that; while `Z_CONSTANT_FLAGS` can be used as r-value now, I'm not sure if it is supposed to.

Reply:
When I remove this line I get segfault, that's why I need to set the value to `ic.value`.

Reply:
Doesn't it work what you did above? ``ZEND_CONSTANT_SET_FLAGS(&ic, 0, PHP_USER_CONSTANT);`